src/manifest.json

```diff
@@ -0,0 +1,19 @@
+{
+  "manifest_version": 2,
+
```

Comment:
@hasnaindev Putting empty lines here is normal or is it a mistake?

Reply:
This is intentional to group similar things and separate dissimilar ones.